README.md

```diff
@@ -56,18 +57,39 @@ The tool has been tested with the following software versions.
 
 ### Executable Installation
 
-In order to install the command line interface, navigate to the root directory of the project and run
+In order to install the command line interface, navigate to the root directory of the project and run the following command.
 
 ```bash
 cabal new-install exe:haskell-src-transformations
 ```
 
-You can also run the executable directly without installing it first.
+The command above builds the executable pattern matching compiler's command line interface and creates a symbolic link in `~/.cabal/bin`.
+To test whether the installation was successful, make sure the `~/.cabal/bin` is in your `PATH` environment variable and run the following command.
+
+```
+haskell-src-transformations --help
+```
+
+### Running without Installation
+
+If you want to run the command line interface without installing the package on your machine (e.g., for debugging purposes), execute the following command in the root directory of the project instead of the `haskell-src-transformations`.
 
 ```bash
 cabal new-run exe:haskell-src-transformations -- [options...]
 ```
 
+The two dashes are needed to separate the arguments to pass to the pattern matching compiler from Cabal’s arguments.
```

Comment:
```suggestion
The two dashes are needed to separate the arguments to pass to the pattern matching compiler from Cabal's arguments.
```
Very minor change for consistency (but we could also use **’** instead of **'** everywhere in the documentations).

Reply:
Good catch. I have no idea how I typed that character in the first place :joy: